rero_ils/config.py

```diff
@@ -795,9 +794,9 @@ def _(x):
         max_result_window=10000,
         search_factory_imp='rero_ils.query:organisation_search_factory',
         read_permission_factory_imp=can_access_organisation_records_factory,
-        create_permission_factory_imp=can_create_organisation_records_factory,
-        update_permission_factory_imp=can_update_organisation_records_factory,
-        delete_permission_factory_imp=can_delete_organisation_records_factory,
+        create_permission_factory_imp=is_system_librarian_organisation_record_factory,
+        update_permission_factory_imp=is_system_librarian_organisation_record_factory,
```

Comment:
I think the librarian needs to have the update permissions of `cipo`. Maybe the excel file of permissions is not up to date. We can have `cipo` at the library level. another point for the PO

Reply:
I create a special permission for `update`.